app/controllers/rentals_controller.rb

```diff
@@ -0,0 +1,72 @@
+class RentalsController < ApplicationController
+
+  RENTAL_LIMIT = 7 #days
+
+  def check_out
+    rental = Rental.new(rental_params)
+    rental.check_out = DateTime.now
+    rental.due_date = rental.check_out + RENTAL_LIMIT
```

Comment:
Moving the rental constant would probably also lead to moving this logic in the model as well